app/controllers/concerns/feature_flagged_concern.rb

```diff
@@ -0,0 +1,9 @@
+module FeatureFlaggedConcern
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def feature_flagged(config_key, **kwargs)
+      before_action(**kwargs) { render_not_found unless IdentityConfig.store.send(config_key) }
```

Comment:
we do this very often! I like the idea of DRY-ing it

Reply:
I'm slightly wary of the dynamic `send` 😬

Reply:
> I'm slightly wary of the dynamic `send` 😬

Any ideas for an alternative? Allowlist? Method on `IdentityConfig` like `IdentityConfig.enabled?(key)` ?

Reply:
Since it's a struct you can use brackets:

`IdentityConfig.store[:feature_name_symbol]`

Reply:
Ope, brackets throws an error but apparently `.dig` does not!
```
IdentityConfig.store.dig(:feature_symbole_name)
```